pkg/services/searchV2/extract/testdata/empty-datasource-variable-info.json

```diff
@@ -7,8 +7,8 @@
   ],
   "datasource": [
     {
-      "uid": "default.uid",
-      "type": "default.type"
+      "uid": "${sqllite}",
+      "type": "sqlite-datasource"
```

Comment:
Are these fallback values if no references are found (for the datasouce)?

Reply:
@ArturWierzbicki recently added code to resolve the template variables into real datasources https://github.com/grafana/grafana/pull/51587 so we can use that for security constraints.

However that code returns nothing if the variable does not resolve... that is less then great since we still want to know what type datasource it is, and should likely show a warning that it did not resolve